### ISSUE TITLE
Sim4Life service versioning

### ISSUE THREAD

**Issue by SCA-ZMT** [ITISFoundation/osparc-issues#1404]
## Conclusions from the working group

- All s4l-based services get identical version numbers
  - `sim4life`
  - `isolve`
  - `JupyterLab`
  - `python runner`
- For now, version numbers will be `8.0.0`, `8.0.1`, `8.0.2`. This requires the Desktop Versioning to be changed because the web will release more often
- The services will be named `sim4life 8.0` (next minor release `sim4life 8.2`) and user will be able to select patch versions
- We will buffer the latest official desktop version and the latest patch
- The Dashboard in the services tab will show all versions similar to the Tutorials in a list with headers: 8.0. 8.2, 9.0
- Since the service key will change from 8.0 to 8.2 we need to improve the updating mechanism in the frontend (first example will be moving from current `lite` to `lite 8.0`)

**Comment by mguidon** [comment]
After discussion with @eofli, we do not need to group the services by version. They can all got into one service. That reduces the problem to making services compatible among minor and major versions and among service keys.

**Comment by drniiken** [comment]
As discussed in the pre-planning, we need a good "DEFINITION OF DONE" for this case.
We/Team can maybe have a chat later this week or next week.
Thx, best, Nik

**Comment by pcrespov** [comment]
![Image](https://github.com/user-attachments/assets/006227f7-58a3-42e9-9ff4-c6191fbc2822)

**Comment by mguidon** [comment]
As discussed, merging of the tiles will not be done programatically but through retagging/labeling of the corresponding (entrypoint-) images. This requires:
- Retag all `isolve`, ` jupyter-smash`, `s4l-python-runners` and `sim4life` master services that are listed in the registry-syncer config (lets first check if tag and "key" label is enough (I dont think we need the "compose-spec" as well)
- Update the list of images to sync
- For all `key:version` pairs above copy all entries in the following tables:
  - `resource_tracker_pricing_plan_to_service`
  - `services_access_rights`
  - `services_specifications`
  - `services_vendor_secrets`
  - `user_preferences_user_service (I think this one can safely be omitted)
- Finally, update the following table:
  - `services_compatibility`

@pcrespov I guess the db part is trivial to script once we are sure the list above is complete?

**Comment by pcrespov** [comment]
## [MartinKippenberger Sprint](https://github.com/ITISFoundation/osparc-issues/blob/master/reviews/20241028_MartinKippenberger.md)

### completed
- https://github.com/ITISFoundation/osparc-simcore/pull/6479
- Discussed design of tags sharing and policies on how to deal with tags that are part of a shared project

### open
- Complete implementation of the API to share tags
- Complete implementation of tags in services 
- List services grouped by tags

**Comment by pcrespov** [comment]
## [EventHorizon Sprint](https://github.com/ITISFoundation/osparc-issues/blob/master/reviews/20250110_EventHorizon.md)

### completed
-  tags can be shared

### open

- list services grouped by tags  (use case: bundles, i.e. group different services versions e.g. s4l family of services like isolve, jupyter, ...)

**Comment by mguidon** [comment]
Closing in favor of https://github.com/ITISFoundation/osparc-issues/issues/1821
